### ISSUE TITLE
HUD Update

### ISSUE THREAD

**Issue by CoreyMeyerCin** [CoreyMeyerCin/TopDungeon#41]
I think we need to get rid of our old placeholder menu, we can use the code and reformat it some to make something much nicer.

I believe we should have a lot of the stats display on the screen. @JeremyPlott I think you said you were looking into an exp bar. Would we be able to the do same thing with HP?

**Comment by JeremyPlott** [comment]
Definitely, the code that handles updating the bars isn't very complicated.
